### PR TITLE
Announce Merbridge 0.7.0 Relesse notes

### PR DESCRIPTION
**Pull request by kebe7jun**

### CODE REVIEW

File: content/zh/blog/releases/release-0.7.0.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+title: "发布 0.7.0"
+linkTitle: "发布 0.7.0"
+date: 2022-07-20
+description: >
+  0.7.0 版本更新内容。
+---
+
+**我们很高兴地发布 Merbridge 0.7.0！**
+
+Merbridge 主要的工作集中在两个方面。
```
Comment (windsonsea) [inline content/zh/blog/releases/release-0.7.0.md]:
```suggestion
本次更新主要体现在两个方面：
```

File: content/zh/blog/releases/release-0.7.0.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+一方面是使用 `tc`（Traffic Control）代替 XDP 来做容器的出入口的流量处理。其主要原因是之前所使用的 XDP Generic 模式存在一些问题，且不推荐生产使用。
```
Comment (windsonsea) [inline content/zh/blog/releases/release-0.7.0.md]:
```suggestion
一方面是使用 `tc`（Traffic Control）代替了 XDP 来做容器的出入口的流量处理。其主要原因是之前所使用的 XDP Generic 模式存在一些问题，且不推荐生产使用。
```

**Review by windsonsea — COMMENTED** [review]
May need to add a blank line after each para.

File: content/zh/blog/releases/release-0.7.0.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+date: 2022-07-20
+description: >
+  0.7.0 版本更新内容。
+---
+
+**我们很高兴地发布 Merbridge 0.7.0！**
+
+本次更新主要体现在两个方面：
+一方面是使用 `tc`（Traffic Control）代替了 XDP 来做容器的出入口的流量处理。其主要原因是之前所使用的 XDP Generic 模式存在一些问题，且不推荐生产使用。
+另一方面，在 Kuma 工程师 [@bartsmykla](https://github.com/bartsmykla) 的帮助下，Merbridge 增加了对 Kuma 的支持，可以在 Kuma 中使用和 Istio 相同的能力。
```
Comment (kebe7jun) [inline content/zh/blog/releases/release-0.7.0.md]: @windsonsea @Michelle951  这里也怎么重新组织下？

File: content/zh/blog/releases/release-0.7.0.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+linkTitle: "发布 0.7.0"
+date: 2022-07-20
+description: >
+  0.7.0 版本更新内容。
+---
+
+**我们很高兴地发布 Merbridge 0.7.0！**
+
+本次更新主要体现在两个方面：
+一方面是使用 `tc`（Traffic Control）代替了 XDP 来做容器的出入口的流量处理。其主要原因是之前所使用的 XDP Generic 模式存在一些问题，且不推荐生产使用。
```
Comment (windsonsea) [inline content/zh/blog/releases/release-0.7.0.md]:
```suggestion
1. 使用 `tc`（Traffic Control）代替了 XDP 做容器出入口的流量处理。其主要原因是之前所使用的 XDP Generic 模式存在一些问题，且不推荐生产使用。
```

File: content/zh/blog/releases/release-0.7.0.md (hunk 3 above)
Comment (windsonsea) [inline content/zh/blog/releases/release-0.7.0.md]:
```suggestion
2. Merbridge 增加了对 Kuma 的支持，可以像在 Istio 一样，在 Kuma 中使用 Merbridge 的所有能力。这个功能多亏了 Kuma 工程师 [@bartsmykla](https://github.com/bartsmykla) 的不懈努力，谢谢。
```

**Review by windsonsea — COMMENTED** [review]
Giving a bullet may look better.

File: content/zh/blog/releases/release-0.7.0.md (hunk 4 above)
Comment (Michelle951) [inline content/zh/blog/releases/release-0.7.0.md]:
```suggestion
1. 使用 `tc`（Traffic Control）代替 XDP 来做容器的出入口流量处理，规避 XDP Generic 模式存在的问题，并且可用于生产环境。
```

File: content/zh/blog/releases/release-0.7.0.md (hunk 3 above)
Comment (Michelle951) [inline content/zh/blog/releases/release-0.7.0.md]:
```suggestion
2. 增加对 Kuma 的支持，可以在 Kuma 中使用和 Istio 相同的能力。感谢 Kuma 工程师 [@bartsmykla](https://github.com/bartsmykla) 的贡献。
```

File: content/zh/blog/releases/release-0.7.0.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+另一方面，在 Kuma 工程师 [@bartsmykla](https://github.com/bartsmykla) 的帮助下，Merbridge 增加了对 Kuma 的支持，可以在 Kuma 中使用和 Istio 相同的能力。
+
+其它改动，请参考：[Merbridge 0.7.0](https://github.com/merbridge/merbridge/releases/tag/0.7.0)
```
Comment (Michelle951) [inline content/zh/blog/releases/release-0.7.0.md]:
```suggestion
有关其他更新内容，请参考：[Merbridge 0.7.0](https://github.com/merbridge/merbridge/releases/tag/0.7.0)
```

**Comment by kebe7jun** [comment]
@Michelle951 @windsonsea Approve this pr plz~

**Review by windsonsea — APPROVED** [review]
looks good to me.

**Pull request merged**